### PR TITLE
Fix breadcumb navbar

### PR DESCRIPTION
**Pull request by irina060981**
First part of the issue - https://github.com/alpheios-project/alpheios_nemo_ui/issues/74
Fix Breadcumb navbar

**xs**
image

**sm**
image

**md**
image

**lg**
image

I decided to solve this issue by two steps - it is the first.

### CODE REVIEW

File: alpheios_nemo_ui/data/assets/scss/breadcumb-navbar.scss [hunk 1]
```diff
@@ -0,0 +1,86 @@
+.breadcumb-navbar {
+    overflow: hidden;
+    display: block;
+    text-align: center;
+
+    .nav-link-current-text {
+        font-size: 14px;
```
Comment (balmas) [inline alpheios_nemo_ui/data/assets/scss/breadcumb-navbar.scss]: can we use variables here? Style guide defines large label as 14px regular sans and then says that this text should be label label on Mobile and H4 on Desktop. So I think we need a new $label-large variable, and then we could have the base style for mobile be :  `font-size: $label-large; font-family: $font-family-sans;` and  override for media-breakpoint-up-lg to `font-size: $h4-font-size; font-family $font-family-serif`

File: alpheios_nemo_ui/data/assets/scss/breadcumb-navbar.scss [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+            top: 3px;
+        }
+    }
+
+    .navbar-section-ref {
+        display: flex;
+        text-align: center;
+        padding: 0.5rem 0;
+        
+        .nav-link {
```
Comment (balmas) [inline alpheios_nemo_ui/data/assets/scss/breadcumb-navbar.scss]: as above for the title, can we use variables here? Style guide defines small label as 12px regular sans and then says that this text should be small label on Mobile and body on Desktop. So I think we need a new $label-large variable, and then we could have the base style for mobile be : font-size: $label-small; font-family: $font-family-sans; and override for media-breakpoint-up-lg to font-size: $base-font-size; font-family $font-family-sans

File: alpheios_nemo_ui/data/assets/scss/alpheios.scss [hunk 3]
```diff
@@ -8,6 +8,10 @@
 @import "common";
 
 @import "design-commons";
+
 @import "help-popup";
+@import "main-navbar";
+@import "breadcumb-navbar";
```
Comment (balmas) [inline alpheios_nemo_ui/data/assets/scss/alpheios.scss]: minor typo in filename -- breadcumb should probably be breadcrumb

**Comment by irina060981** [comment]
a) I have rearranged styles - add style-guide mixins.
b) check styles for navbar, breadcumb-navbar, top-level-navbar and help-popup according to the style-guide
c) add popup for bibliog popup
**desktop**
image

**mobile**
image

**Pull request merged**